environments/key4hep-desy-release/spack.yaml

```
@@ -0,0 +1,21 @@
+spack:
+  config:
+    install_tree:
+      root: $CVMFS_INSTALL_DIR/spackages
+      projections:
+        all: ${PACKAGE}/${VERSION}/${target}-${os}-${COMPILERNAME}${COMPILERVER}-opt/${HASH}
+        build_type=Debug: ${PACKAGE}/${VERSION}/${target}-${os}-${COMPILERNAME}${COMPILERVER}-dbg/${HASH}
+    module_roots:
+      lmod: $CVMFS_INSTALL_DIR/spackages/modules
+      tcl: $CVMFS_INSTALL_DIR/spackages/modules
+
+  view: false
+  # More configuration in other config files
+  include:
+  - desy-packages.yaml
+  modules:
+    enable: []
+  concretization: together
+  specs:
+    - key4hep-stack
+    - fccanalyses@commit.df65dc1059691029ac21da51cdbea37d35249681
```

I guess this comes from a `spack add`?
```suggestion
  specs:
    - key4hep-stack
    - fccanalyses@commit.df65dc1059691029ac21da51cdbea37d35249681
```

---

In this case I put them here deliberately, mainly to have the exact version for `fccanalyses` under version control. Adding `key4hep-stack` as well, just lets me skip the `spack add` (and also `spack concretize` in the initial setup) and directly do a `spack install` of the environment. Obviously, once there is an existing installation, and the environment should pick up new changes re-concretizing will become necessary again.

---

Made a new patch release for fccanalyses: https://github.com/HEP-FCC/FCCAnalyses/releases/tag/v0.3.7

---

Added that as well to this PR.